crates/utils/re_mcap/src/layers/ros2.rs

```diff
@@ -52,6 +53,7 @@ impl MessageLayer for McapRos2Layer {
             }
             "sensor_msgs/msg/PointCloud2" => Box::new(PointCloud2MessageParser::new(num_rows)),
             "sensor_msgs/msg/NavSatFix" => Box::new(NavSatFixMessageParser::new(num_rows)),
+            "rcl_interfaces/msg/Log" => Box::new(LogMessageParser::new(num_rows)),
```

Comment:
This message list would be easier to scan if sorted alphabetically (and less likely to produce merge conflicts)